server/src/domain/job/job.service.ts

```diff
@@ -228,4 +229,12 @@ export class JobService {
         break;
     }
   }
+
+  private validateCronExpression(cronExpression: string) {
```

Comment:
I think we need to move this into a repository. We don't need to be referencing importing/using `cron` directly in the domain.